radio/src/datastructs.h

```diff
@@ -980,7 +980,11 @@ PACK(struct RadioData {
   NOBACKUP(int8_t backlightMode);
   NOBACKUP(TrainerData trainer);
   NOBACKUP(uint8_t view);            // index of view in main screen
-  NOBACKUP(int8_t buzzerMode:2);    // -2=quiet, -1=only alarms, 0=no keys, 1=all
+#if defined(BUZZER)
+  NOBACKUP(int8_t buzzerMode:2);    // -2=quiet, -1=only alarms, 0=no keys, 1=all (only used on AVR radios without audio hardware)
```

Comment:
Your first comment is wrong, the code you pasted is not complete. First `#if` has effectively a condition `#if defined(AUDIO) && defined(BUZZER)`, the `#elif` is only executed if `AUDIO` is not defined and `BUZZER` is defined.

The only cleanup possible is macros in `adudio_arm.h` where `BUZZER` is used, but is never defined.